codec-http2/src/main/java/io/netty/handler/codec/http2/hpack/Decoder.java

```diff
@@ -0,0 +1,568 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
```

Comment:
Is this the right way to do this? can we just append multiple copyrights to the same apache2 license statement, or do we need the `The Netty Project licenses this file to you` prefix?

```java
/*
 * Copyright 2014 Twitter, Inc.
 * Copyright 2015 The Netty Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
...
* /
```